build.rs

```diff
@@ -62,6 +62,27 @@ fn build_nasm_files() {
   rerun_dir("src/ext/x86");
 }
 
+#[cfg(feature = "asm")]
+fn build_asm_files() {
+  use std::fs::File;
+  use std::io::Write;
+  let out_dir = env::var("OUT_DIR").unwrap();
+  {
+    let dest_path = Path::new(&out_dir).join("config.h");
+    let mut config_file = File::create(dest_path).unwrap();
+    config_file.write(b" #define ARCH_AARCH64 1\n").unwrap();
+    config_file.write(b" #define ARCH_ARM 0\n").unwrap();
+    config_file.write(b" #define CONFIG_LOG 1 \n").unwrap();
+    config_file.write(b" #define HAVE_ASM 1\n").unwrap();
+  }
+  cc::Build::new()
+    .files(&["src/arm/64/mc.S"])
+    .include(".")
+    .include(&out_dir)
+    .define("PRIVATE_PREFIX", "rav1e_")
+    .compile("rav1e-aarch64");
+}
```

Comment:
Currently, cc-rs does not emit `rerun-if-` lines for sources or environment variables.
We should at least rebuild if the assembly is modified.
```suggestion
  rerun_dir("src/arm");
}
```

Reply:
maybe also open an issue about it.

Reply:
That issue is https://github.com/alexcrichton/cc-rs/issues/230